app/modules/import-export/runners/import/site-settings.php

```diff
@@ -162,6 +178,74 @@ public function import_theme( array $data ) {
 		return $result;
 	}
 
+	private function import_experiments( array $data ) {
+		if ( empty( $data['site_settings']['experiments'] ) ) {
+			return null;
+		}
+
+		$experiments_data = $data['site_settings']['experiments'];
+		$experiments_manager = Plugin::$instance->experiments;
+		$current_features = $experiments_manager->get_features();
+
+		$this->save_previous_experiments_state( $current_features );
+
+		$imported_count = 0;
+
+		foreach ( $experiments_data as $feature_name => $feature_data ) {
+			if ( ! isset( $current_features[ $feature_name ] ) ) {
+				continue;
+			}
+
+			$current_feature = $current_features[ $feature_name ];
+
+			if ( ! $current_feature['mutable'] ) {
+				continue;
+			}
+
+			$new_state = $feature_data['state'];
+			if ( ! in_array( $new_state, [ 'default', 'active', 'inactive' ], true ) ) {
```

Comment:
Use consts in elementor/core/experiments/manager.php

STATE_DEFAULT
STATE_ACTIVE
STATE_INACTIVE